DB/Vagrantfile

```diff
@@ -45,7 +46,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
   # config.vm.synced_folder "../data", "/vagrant_data"
-  config.vm.synced_folder "puppet", "/etc/puppet"
+  config.vm.synced_folder "../../Backend", "/home/vagrant/Backend"
```

Comment:
Ok, but I think that breaks automated deployment. Can you keep that change in your local copy?